boards/xtensa/intel_s1000_crb/dts_fixup.h

```diff
@@ -15,15 +15,15 @@
 #define DT_CODEC_NAME			\
 	DT_SNPS_DESIGNWARE_I2C_80400_TI_TLV320DAC_18_LABEL
 
-#define CONFIG_SPI_NOR_SPI_NAME			\
+#define DT_SPI_NOR_SPI_NAME			\
 	DT_SNPS_DESIGNWARE_SPI_E000_SPI_FLASH_0_BUS_NAME
 
-#define CONFIG_SPI_NOR_SPI_FREQ_0		\
+#define DT_SPI_NOR_SPI_FREQ_0		\
 	DT_SNPS_DESIGNWARE_SPI_E000_SPI_FLASH_0_SPI_MAX_FREQUENCY
 
-#define CONFIG_SPI_NOR_DRV_NAME			\
+#define DT_SPI_NOR_DRV_NAME			\
 	DT_SNPS_DESIGNWARE_SPI_E000_SPI_FLASH_0_LABEL
 
-#define CONFIG_SPI_NOR_SPI_SLAVE		\
+#define DT_SPI_NOR_SPI_SLAVE		\
```

Comment:
Interesting choice of the name "slave". The parameter is an address, so please change to an appropriate name.

Reply:
Well.. this name is generated by the python scripts for DTS, i think. Can we take this as a separate exercise? I would suggest you log a GH issue on this so that appropriate people can look into this.

Reply:
No. DTS generates `DT_SNPS_DESIGNWARE_SPI_E000_SPI_FLASH_0_BASE_ADDRESS` and is being redefined as `DT_SPI_NOR_SPI_SLAVE` in DTS fixup.

Reply:
Done